state-loader.md

```diff
@@ -230,6 +231,23 @@ The `"rlp"` key loads the block information.
           ...
           </k>
 
+    rule <k> load "transaction" : [ (T => [#rlpDecodeTransaction(#parseByteStackRaw(T))]) , _ ]
+          ...
+          </k>
```

Comment:
```suggestion
    rule <k> load "transaction" : [ (T => [#rlpDecodeTransaction(#parseByteStackRaw(T))]) , _ ]
          ...
          </k>
```

Put this all on one line and consider moving it somewhere else.

Reply:
I don't understand why should I move it.

Reply:
Well, it's between the two rules which handle the decoded transaction. So this rule does the decoding, but the two rules surrounding it actually handle the decoded transaction. Just seems a little out of place I guess, but up to you.